server/channels/app/file_test.go

```diff
@@ -361,8 +373,10 @@ func TestCopyFileInfos(t *testing.T) {
 	info1, err := th.App.DoUploadFile(th.Context, time.Date(2007, 2, 4, 1, 2, 3, 4, time.Local), teamID, channelID, userID, filename, data, true)
 	require.Nil(t, err)
 	defer func() {
-		th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, info1.Id)
-		th.App.RemoveFile(info1.Path)
+		err := th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, info1.Id)
+		require.NoError(t, err)
+		appErr := th.App.RemoveFile(info1.Path)
+		require.Nil(t, appErr)
```

Comment:
That errors with `        	Error:      	Expected nil, but got: &model.AppError{Id:"api.file.remove_file.app_error", Message:"Unable to remove the file.", DetailedError:"", RequestId:"", StatusCode:500, Where:"RemoveFile", SkipTranslation:false, params:map[string]interface {}(nil), wrapped:(*errors.withStack)(0xc005b79560)}
`

Can you please take a look?